helix-rest/pom.xml

```diff
@@ -154,6 +154,17 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-jersey2</artifactId>
+      <version>4.1.11</version>
```

Comment:
Why we need these dependencies in the other modules (helix-core and metric-core)? I assume we are just going to use it for helix-rest, no?